src/NECompletion-Tests/NECTestClass.class.st

```diff
@@ -38,7 +38,7 @@ NECTestClass class >> initialize [
 { #category : #initialization }
 NECTestClass >> initialize: aRectangle [ 
 	constantInteger := 15.
-	constantLargeInteger := 1073741824.
+	constantLargeInteger := 1152921504606846976. "(SmallInteger maxVal + 1) on 64-bit"
```

Comment:
Isn't it possible to directly use `SmallInteger maxVal +1`?

Reply:
no, it must be a constant, of course I tried that :-/